<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLbl">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Experiment Information and Setup</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0" columnminimumwidth="0,0,0">
     <item row="0" column="0">
      <widget class="QToolButton" name="proposalBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Edit Proposal Information</string>
       </property>
       <property name="text">
        <string>Proposal</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QToolButton" name="setupBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Edit Instrument Setups</string>
       </property>
       <property name="text">
        <string>Setups</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QToolButton" name="sampleBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Edit Samples</string>
       </property>
       <property name="text">
        <string>Samples</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="ValueLabel" name="valueLabel_5">
       <property name="key" stdset="0">
        <string>exp.remark</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Users</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Local Contact</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ValueLabel" name="valueLabel">
       <property name="key" stdset="0">
        <string>exp.proposal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ValueLabel" name="valueLabel_3">
       <property name="key" stdset="0">
        <string>exp.users</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ValueLabel" name="valueLabel_6">
       <property name="key" stdset="0">
        <string>sample.samplename</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ValueLabel" name="valueLabel_4">
       <property name="key" stdset="0">
        <string>exp.localcontact</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Detectors</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Scans</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ValueLabel" name="setupLabel"/>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="ValueLabel" name="detLabel">
       <property name="text">
        <string>(exp.detlist)</string>
       </property>
       <property name="key" stdset="0">
        <string>exp.detlist</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="ValueLabel" name="envLabel">
       <property name="text">
        <string>(exp.envlist)</string>
       </property>
       <property name="key" stdset="0">
        <string>exp.envlist</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ValueLabel" name="valueLabel_2">
       <property name="key" stdset="0">
        <string>exp.title</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QToolButton" name="detenvBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Edit Detector and Scan Environments</string>
       </property>
       <property name="text">
        <string>Environments</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QToolButton" name="remarkBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Add a Remark</string>
       </property>
       <property name="text">
        <string>Remark</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/jump</normaloff>:/jump</iconset>
   </property>
   <property name="text">
    <string>Move...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionFix">
   <property name="text">
    <string>Fix...</string>
   </property>
  </action>
  <action name="actionRelease">
   <property name="text">
    <string>Release</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/emergency</normaloff>:/emergency</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPlotHistory">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/find</normaloff>:/find</iconset>
   </property>
   <property name="text">
    <string>Plot history...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueLabel</class>
   <extends>QLabel</extends>
   <header>nicos.guisupport.display</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
